sdmetrics/visualization.py

```diff
@@ -1,14 +1,45 @@
 """Visualization methods for SDMetrics."""
 
+from functools import wraps
+
 import pandas as pd
 import plotly.express as px
 import plotly.figure_factory as ff
+import plotly.io as pio
 from pandas.api.types import is_datetime64_dtype
 
 from sdmetrics.reports.utils import PlotConfig
 from sdmetrics.utils import get_missing_percentage, is_datetime
 
 
+def set_plotly_config(function):
+    """Set the ``plotly.io.renders`` config according to the environment.
+
+    Configure the rendering settings based on the environment in which the plot is generated
+    to ensure the image rendering with a stable engine. For other environments, like
+    ``Jupyter Notebooks``, select the ``iframe`` rendering engine otherwise leave the default
+    one.
+    """
+    @wraps(function)
+    def wrapper(*args, **kwargs):
+        renderers = list(pio.renderers)
+        try:
+            # Lazy import IPython
+            from IPython import get_ipython
```

Comment:
does this import fail if the person isn't on jupyter notebbok?

Reply:
This import can fail if you don't have `ipython` or `jupyter` installed.